### ISSUE TITLE
Error when adding class to <ol>

### ISSUE THREAD

**Issue by julianomoreira** [arve0/markdown-it-attrs#28]
We're getting an error and our site won't compile when we try to add a class {.something} to an ordered list.

**Comment by arve0** [comment]
Thanks for the feedback. Can you be more specific? What does the error say? Please add an example and/or jsfiddle also, with expected input and output.

**Comment by arve0** [comment]
Input:
```md
1. one
2. two
{.c}
```

Error: 
```
TypeError: undefined is not an object (evaluating 'e.attrJoin')
```

**Comment by arve0** [comment]
Thanks for reporting. Fixed in `v0.7.0`

**Comment by julianomoreira** [comment]
Sorry I didn't paste an example but you were able to replicate it. Thank you!
